packages/tackle-box/lib/components/Input/Input.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from "react";
```

Comment:
suggestion: Since the component is TextInput, should we name this file TextInput.tsx?